kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/NetworkConfigurationServiceImpl.java

```diff
@@ -196,7 +197,7 @@ public void activate(ComponentContext componentContext, Map<String, Object> prop
             logger.debug("Got null properties...");
         } else {
             logger.debug("Props...{}", properties);
-            this.properties = new HashMap<>(properties);
+            this.properties = discardModifiedNetworkInterfaces(new HashMap<>(properties));
```

Comment:
Following the other comment:

```suggestion
            this.properties.remove(MODIFIED_INTERFACE_NAMES);
```

Reply:
As explained [here](https://github.com/eclipse/kura/pull/4242#discussion_r1023658145) this is not a good idea. Consider this as "Resolved".